softee220/as_build_vector_db.py

```
@@ -0,0 +1,159 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+get_ipython().system('pip install -r ../requirements.txt')
```

Ruff (0.13.2)

Undefined name `get_ipython`

(F821)